api/src/pcapi/core/search/backends/algolia.py

```diff
@@ -760,6 +761,35 @@ def _clean_processing_queue(self, processing_queue: str, originating_queue: str)
                     extra={"queue": originating_queue, "processing_queue": processing_queue},
                 )
 
+    def seach_offers_ids(self, query: str = "", filters: str = "", count: int = 20) -> list[int]:
+        ids: list[int] = []
+        params: dict[str, str | int] = {"filters": filters}
+        for page, _count in enumerate(
+            range(
+                0,
+                count,
+            )
+        ):
+            # handle alogolia pagination of results
```

Comment:
non mais alo, quoi !
```suggestion
            # handle algolia pagination of results
```